src/providers/aws.rs

```diff
@@ -228,6 +230,13 @@ impl<P> Launcher<P> {
         self
     }
 
+    /// The machines spawned on this launcher will have 
+    /// ports open to the public Internet.
+    pub fn open_ports(&mut self) -> &mut Self {
```

Comment:
I think we want this for azure as well, but since Azure is new it's not a requirement for this PR.

Reply:
I looked briefly at doing the same for azure, but it is somewhat more complicated (I couldn't find an equivalent of `172.31.0.0/16`), so I left it for now.